ui/src/data/source.tsx

```diff
@@ -1606,19 +1606,35 @@ function isInternalAttribute(attribute: string): boolean {
 }
 
 export function literalFromDataValue(value: DataValue): tanagra.Literal {
-  let dataType = tanagra.DataType.Int64;
-  if (typeof value === "string") {
+  let dataType: DataType;
+  if (typeof value === "bigint") {
+    dataType = tanagra.DataType.Int64;
+  } else if (typeof value == "number") {
+    dataType = Number.isInteger(value)
```

Comment:
Is this required? The type should reflect the type in the database and numbers should only be used for DOUBLEs now.

Reply:
Wasn't sure, so had kept it. removing now

Reply:
Looks like it's still here?

Reply:
my bad, one change was left out during push from local